vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -205,7 +205,8 @@ class LedgerImpl : public Ledger {
   void HasSufficientBalanceToReconcile(
       HasSufficientBalanceToReconcileCallback callback) override;
 
-  void GetRewardsInternalsInfo(RewardsInternalsInfoCallback callback) override;
+  void GetRewardsInternalsInfo(const std::string& declared_geo,
```

Comment:
Instead of routing through bat-native-ledger (which currently doesn't really "know" the user's declared geo), would it be possible to leave this function as-is, and create a new message handler in the internals WebUI code specifically for "declared geo"? It could just read a pref, similar to [brave_rewards_api.cc](https://github.com/brave/brave-core/blob/a361584c5c1000d15ed54fe10c3084637910dbfe/browser/extensions/api/brave_rewards_api.cc#L660).

Reply:
Yeah, was definitely struggling with whether to use the pref or plumb it through. Fixed it up to use the pref, that makes things much more concise too. Thanks!